test/functional-portable/corerp/cloud/resources/testdata/corerp-resources-extender-aws-s3-recipe.bicep

```diff
@@ -1,15 +1,16 @@
-import radius as rad
+provider radius
 
 param bucketName string
 param creationTimestamp string
 param awsAccountId string
 param awsRegion string
 param registry string 
 param version string
+param location string = 'global'
 
 resource env 'Applications.Core/environments@2023-10-01-preview' = {
   name: 'corerp-resources-extenders-aws-s3-recipe-env'
-  location: 'global'
+  location: location
```

Comment:
Is there a reason why we need to specify the location here? We made a change to make location optional in Bicep for our resources.

Reply:
If location is not showing as optional, then that's a regression and we need to fix it.

Reply:
Location has as required flag in the types files so this might be a regression in the generation logic. Will look into that and fix it. But our docs also have location as a required field so that probably needs to get updated too: https://docs.radapp.io/reference/resource-schema/overview/

Reply:
Whether or not it's required depends on which client you're talking about. The Radius API requires location to be set. Bicep will default it for you. One of the ways we tweak the Bicep experience to be simpler.

Reply:
This was a casing bug in the generator. Fixed as part of the PR